### ISSUE TITLE
Option to edit output photo name

### ISSUE THREAD

**Issue by edencakir** [npomfret/react-native-image-tools#7]
Saved photos are always saved with a prefix 'rnimagetools', is there a way to avoid that?

**Comment by srameshr** [comment]
@edencakir Are you able to save the images? For me, as soon as I click the done button the app crashes on <b>IOS</b>. Could you share me your implementation?

**Comment by edencakir** [comment]
Ive only implemented for Android.

**Comment by edencakir** [comment]
@srameshr can you post the stacktrace of the error?

**Comment by srameshr** [comment]
@edencakir I fixed it. It was permissions issue. BTW whats the increase in file size when u download the SDK? What will be the file size when you proguard compress it?

**Comment by edencakir** [comment]
Nope, it's not a concern for my project. SDK is standard and can't do anything about it - unrelated to this repo.

**Comment by srameshr** [comment]
@edencakir I understand that. I just want to know whats the size the editor sdk adds to ur app.

**Comment by edencakir** [comment]
APK size is 17 MB, I'm not sure how much of it is from SDK.

**Comment by npomfret** [comment]
Np plans to pass in an option for naming the photos.  it easy enough to change them using `rn-fetch-blob`.  Or submit a PR if you desperately want the feature.

**Comment by srameshr** [comment]
@npomfret Its not immediate. Passing tools option to the editor will be an awesome addition.
